test/docfx.Test/e2e/E2ETest.cs

```diff
@@ -274,7 +274,7 @@ private static string ToSafePathString(string value)
 
                             foreach (var (path, content) in commit.Files)
                             {
-                                var blob = repo.ObjectDatabase.CreateBlob(new MemoryStream(Encoding.UTF8.GetBytes(content?.Replace("\r", "") ?? "")));
```

Comment:
we may still need this one if test content is multi-line and people have different line ending configs.